doc/INSTALL.md

```diff
@@ -42,11 +42,39 @@ cp dataactvalidator/config/example_object_class.csv dataactvalidator/config/obje
 cp dataactvalidator/config/example_program_activity.csv dataactvalidator/config/program_activity.csv```
 ```
 
+### Load or Update Domain Data
```

Comment:
Would this be better placed after the Docker or personal DB setup?

Better question: Would it make more sense inside the actual setup docs?